okta/user.go

```diff
@@ -29,6 +29,18 @@ func assignAdminRolesToUser(u string, r []interface{}, c *okta.Client) error {
 	return nil
 }
 
+func assignGroupsToUser(u string, g []interface{}, c *okta.Client) error {
+	for _, group := range g {
+		_, err := c.Group.AddUserToGroup(group.(string), u)
```

Comment:
I am not sure how much speed matters here but making these requests serial might get slow. Although I would imagine it will be a bit before we notice it but maybe it is worth putting a pattern in place now or soon for parallel requests. I think this is simple enough to implement goroutines and barely notice the added complexity, thoughts?